pipeline.Rproj

```diff
@@ -0,0 +1,15 @@
+Version: 1.0
```

Comment:
hey Agi, why are we adding this here? (meaning the rproj file)

Reply:
Ah sorry 😓  left over from opening it up in Rstudio. I'll remove it